plugins/rbac/dev/index.tsx

```diff
@@ -90,12 +90,11 @@ class MockRBACApi implements RBACAPI {
     return mockPermissionPolicies;
   }
 
-  async deletePolicy(
+  async deletePolicies(
     _entityRef: string,
-    _permission: string,
     _policies: Policy[],
-  ): Promise<number> {
-    return 204;
+  ): Promise<Response | { status: string; statusText: string }> {
+    return { status: 204, statusText: 'Deleted Successfully' } as Response;
```

Comment:
in dev mode, I see below

<img width="645" alt="image" src="https://github.com/janus-idp/backstage-plugins/assets/5129024/23211a41-6cba-4b7e-b4f7-1cd30e29d1ca">